src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -157,7 +157,7 @@ internal class SqlServerFhirDataStore : IFhirDataStore, IProvideCapability
             }
 
             // Ignore input resource version to get latest version from the store.
-            var existingResources = (await GetAsync(resources.Select(r => r.Wrapper.ToResourceKey(true)).Distinct().ToList(), cancellationToken)).ToDictionary(r => r.ToResourceKey(true), r => r);
+            var existingResources = (await GetAsync(resources.Select(r => r.Wrapper.ToResourceKey(true)).Distinct().ToList(), true, cancellationToken)).ToDictionary(r => r.ToResourceKey(true), r => r);
```

Comment:
NIT: Do you mind adding the parameter name "includeInvisible" just to make it clear why "true" is being used in this statement.

Reply:
Added comments.